src/Command/ConsumerCommand.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Skrz\Bundle\BunnyBundle\Command;
+
+use Bunny\Channel;
+use Bunny\Client;
+use Bunny\Message;
+use Bunny\Protocol\MethodBasicQosOkFrame;
+use Bunny\Protocol\MethodQueueBindOkFrame;
+use Bunny\Protocol\MethodQueueDeclareOkFrame;
+use InvalidArgumentException;
+use Skrz\Bundle\BunnyBundle\Exception\BunnyException;
+use Skrz\Bundle\BunnyBundle\Queue\BunnyConsumerInterface;
+use Skrz\Bundle\BunnyBundle\Queue\BunnyTickingConsumer;
+use Skrz\Bundle\BunnyBundle\Queue\BunnyTickingConsumerInterface;
+use Skrz\Bundle\BunnyBundle\Service\BunnyManager;
+use Skrz\Bundle\BunnyBundle\Service\ContentTypes;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Serializer\SerializerInterface;
+
+/** @author Lukas Senfeld <skrz@senfeld.net> */
+class ConsumerCommand extends Command
+{
+
+	/** @var BunnyConsumerInterface[] */
+	private iterable $consumers;
+	/**
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint
+	 * @var string|null The default command name
+	 */
+	protected static $defaultName = 'bunny:consumer';
+	private BunnyManager $manager;
+	private int $messages = 0;
+	private SerializerInterface $serializer;
+
+	/** @param BunnyConsumerInterface[] $consumers */
+	public function __construct(BunnyManager $manager, SerializerInterface $serializer, iterable $consumers)
```

Comment:
Náhodou by nešlo na $consumers využít splat operator?

Reply:
nesel protoze to hle je SymfonDI otagovany Iterator a tam splat nefunguje, obecne u konstruktoru services se plat pouzivat temer neda